src/panel/direction/DirectionPanel.jsx

```diff
@@ -92,6 +124,16 @@ export default class DirectionPanel extends React.Component {
     Promise.all(poiRestorePromises).then(([ origin, destination ]) => {
       persistentPointState.origin = origin;
       persistentPointState.destination = destination;
+
+      // Show the raw POI name/latlon while waiting for computeRoutes to complete
+      if (origin || destination) {
+        this.setState({
+          originInputText: ' ',
```

Comment:
Why a single space character instead of an empty string like it's already done in the initial state?

Reply:
to avoid opening the suggest while the address is loading (didn't find how to disable it yet)

Reply:
Maybe a less hackish solution would be a dedicated state value, indicating that the form is still initializing and preventing the suggest to appear?